### PR TITLE
Ref #14

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Add bootstrap, test_stats.

### CODE REVIEW

**Comment by EdwardSafford-NOAA** [comment]
This adds the bootstrap_ci code to stats.py and adds a test_stats.py file.

File: src/emcpy/stats.py [hunk 1]
```diff
@@ -247,3 +247,41 @@ def get_linear_regression(x, y):
...
+        level: (float, default=0.95) desired confidence level for CI bounds
+        estimator: (char, default='mean') type of statistic obtained from
+                  the sample (mean or median)
+        nrepl: (integer, default=1000) number of replicates
+
+    Returns:
+        Lower and upper bounds of confidence intervals
+    """
+    if any(_np.isnan(insample)):
+        print('{:s}').format('bootstrap_ci.py: NaN detected. Dropping NaN(s) input prior to bootstrap...')
```
Comment (CoryMartin-NOAA) [inline src/emcpy/stats.py:271]: Is there a reason why to not just use `print('bootstrap_ci.....')`?

File: src/tests/test_stats.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from emcpy.stats import mstats, lregress, ttest, get_weights,\
+    get_linear_regression, bootstrap
+import numpy as np
+
+
+def test_mstats():
```
Comment (CoryMartin-NOAA) [inline src/tests/test_stats.py:6]: thank you for adding additional unit tests!

**Review by CoryMartin-NOAA — APPROVED** [review]
I think it all looks good to me, thanks @EdwardSafford-NOAA for adding this!

**Comment by CoryMartin-NOAA** [comment]
Note to all admins: when doing 'squash and merge', please make sure to give the squash merge commit a name that is more descriptive. Perhaps "Add in bootstrap CI code" or something like that?

File: src/emcpy/stats.py (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply src/emcpy/stats.py:271]: No reason in particular.  I made as few changes as I could to the original code.  I'll be glad to simplify this though.

Comment (CoryMartin-NOAA) [reply src/emcpy/stats.py:271]: I don't think that will be necessary, it works as is and the style tests pass.

Comment (EdwardSafford-NOAA) [reply src/emcpy/stats.py:271]: Fair enough.  I can clean that up if/when other changes need to be made to that file.

**Pull request merged**